frontend/src/views/SendTestMail.vue

```diff
@@ -0,0 +1,81 @@
+<template>
+    <form action="">
+        <div class="modal-card" style="width: auto">
+            <header class="modal-card-head">
+                <p class="modal-card-title">{{ $t('templates.sendtest') }}</p>
+            </header>
+            <section class="modal-card-body">
+                <b-field :label="$t('settings.smtp.toEmail')" label-position="on-border">
+                  <b-input
+                        type="email"
+                        ref="inputTestEmail"
+                        v-model="testEmail"
+                        placeholder='email@site.com'
+                        required>
+                  </b-input>
+                </b-field>
+                <div v-if="errMsg">
+                <b-field class="mt-4" type="is-danger">
+                  <b-input v-model="errMsg" type="textarea"
+                    custom-class="has-text-danger is-size-6" readonly />
+                </b-field>
+              </div>
+            </section>
+            <footer class="modal-card-foot">
+                <b-button
+                    :label="$t('globals.buttons.close')"
+                    @click="$emit('close')" />
+                <b-button
+                    :label="$t('globals.buttons.sendtest')"
+                    :disabled="!isFormValid"
+                    type="is-primary"
+                    @click.prevent="sendTest(data.id, testEmail)"/>
```

Comment:
see other comment below; remove params here?
```suggestion
                    @click.prevent="sendTest"/>
```